src/main/resources/tomcat-eureka-config/dbmigration/registry-changelog-ec-i2b2-integration-config.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!--
+  #%L
+  Eureka! Clinical Registry Service
+  %%
+  Copyright (C) 2017 - 2018 Emory University
+  %%
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+       http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  #L%
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
All of the URLs for the registry entries need to be syntactically valid URLs, even if not all of these services will be running in embedded tomcat.